test/testkit/mocks/backend/deployment.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	otelCollectorImage = "europe-docker.pkg.dev/kyma-project/prod/kyma-otel-collector:0.114.0-main"
+	otelCollectorImage = "europe-docker.pkg.dev/kyma-project/prod/kyma-otel-collector:0.115.0-main"
```

Comment:
Can we maybe take the value from the main.go constants as well to reduce the amount of files to touch?